yarn.lock

```diff
@@ -14282,6 +14282,11 @@ url@^0.11.0:
     punycode "1.3.2"
     querystring "0.2.0"
 
+use-debounce@^7.0.0:
```

Comment:
Is this needed?

Reply:
That is weird. I didn't install it. Is it possible that somehow it installed it when I ran `yarn install` when I cloned the repo?!

Reply:
It's possible (and likely lol) that it might be a mistake on a previous PR I made, and neglected to commit the yarn.lock file. I'm okay merging it in then

Reply:
It's up to you. I can try downgrading it if you want me to.